kvmagent/kvmagent/plugins/tensorfusion/utils.py

```
@@ -0,0 +1,39 @@
+        @LibvirtAutoReconnect
+        def _lookup(conn):
+            return conn.lookupByName(vm_uuid)
+
+        dom = _lookup()
+        return dom.isActive() == 1
+    except libvirt.libvirtError as e:
+        if e.get_error_code() == libvirt.VIR_ERR_NO_DOMAIN:
+            return False
+        logger.warn('is_vm_running: libvirt error checking VM %s: %s' % (vm_uuid, e))
+        return None
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)